config/environments/production.rb

```diff
@@ -40,8 +40,8 @@
   # config.action_dispatch.x_sendfile_header = 'X-Sendfile' # for Apache
   # config.action_dispatch.x_sendfile_header = 'X-Accel-Redirect' # for NGINX
 
-  # Store uploaded files on the local file system (see config/storage.yml for options)
-  config.active_storage.service = :local
+  # Store uploaded files on S3
+  config.active_storage.service = :amazon
```

Comment:
it seems like if an app _has_ uploads we'd virtually always want to use S3, so this seems sensible. I'd only lean against it if setting this service declaration then forced us to set up S3 immediately when starting a new app (that's a time-consuming process and it's better done on an as-needed basis). but if there are no validations here that check for config values if amazon is selected, I'm good with having s3 as the default instead of local in this repo